lib/recurly/plan.php

```diff
@@ -1,5 +1,41 @@
 <?php
-
+/*
+ * class Recurly_Plan
+ * @property string $plan_code Unique code to identify the plan. This code may only contain the following characters: [a-z 0-9 @ - _ .]. Max of 50 characters.
+ * @property string $name Plan name. Max of 255 characters.
+ * @property array of objects $unit_amount_in_cents Array of currency objects, see example below. Max 10000000.
```

Comment:
So this one is showing the limits of the site we use for docs. In PHP land it's a `Recurly_CurrencyList` so this makes me wonder if we should just list the properties and a type but not bother with copying over the descriptions.